src/docs/development/tools/sdk/overview.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Flutter SDK overview
+short-title: Flutter SDK
+description: Flutter libraries and command-line tools.
+---
+
+The Flutter SDK has the packages and command-line tools that you need to develop
+Flutter apps across platforms. To get the Flutter SDK, see [Install][].
+
+## What's in the Flutter SDK
+
+The following is available through the Flutter SDK:
+
+* [Dart SDK][]
+* Heavily optimized, mobile-first 2D rendering engine with
+  excellent support for text
+* Modern react-style framework
+* Rich set of widgets implementing Material Design and iOS styles
+* APIs for unit and integration tests
+* Interop and plugin APIs to connect to the system and 3rd-party SDKs
+* Headless test runner for running tests on Windows, Linux, and Mac
+* [Dart DevTools][] for testing, debugging, and profiling your app
+* `flutter` and `dart` command-line tools for creating, building, testing,
+  and compiling your apps
+
+Note: For more information about the Dart SDK, see its
+[README file][].
+
+## `flutter` command-line tool
+
+The `flutter` CLI tool (`flutter/bin/flutter`) is how developers (or IDEs on behalf of developers) interact with Flutter.
+For more information about `flutter`, see the [`flutter` reference page][].
+
+## `dart` command-line tool
+
+The `dart` CLI tool is available with the Flutter SDK at `flutter/bin/dart`.
+For more information about `dart`, see the [`dart` reference page][].
```

Comment:
It seems like we could skip the "about `flutter`" and "about `dart`" — that we've said those words enough for the context to be clear.